src/fastfield/delete.rs

```diff
@@ -16,7 +15,7 @@ use std::io::Write;
 pub fn write_delete_bitset(
     delete_bitset: &BitSet,
     max_doc: u32,
-    writer: &mut WritePtr,
+    writer: &mut dyn Write,
```

Comment:
Why?

Reply:
It's the better abstraction, anyone with `Write` can use it (in that case the tests)